crates/common/trie/benches/trie_bench.rs

```diff
@@ -0,0 +1,99 @@
+// This code is originally from https://github.com/citahub/cita_trie/ and
+// modified to suit our needs, and to have a baseline to benchmark our own
+// trie implementation against an existing one.
```

Comment:
It might be a good idea to mention:
1. The commit used as reference;
2. The license (which seems to be Apache 2) that allows us to use it.